redaxo/src/core/backend.php

```diff
@@ -200,6 +200,8 @@
 // ----- Prepare AddOn Pages
 if (rex::getUser()) {
     rex_be_controller::appendPackagePages();
+
+    rex_api_function::handleCall();
```

Comment:
Bin mir unsicher ob wir damit BC probleme aufmachen.

wo genau passiert die umleitung,  wenn keine page gesetzt ist? Ggf. Können wir auch einfach nur `rex_api_function::handleCall` aufrufen, wenn keine gültige/oder gar keine page gegeben ist und bei valider page den aufrufzeitpunkt beibehalten?

Reply:
Ursache sollte folgende sein (Ist schon etwas her)
Siehe redaxo/src/core/lib/be/controller.php => Zeile 377: 
Z. 375: $page = self::getCurrentPageObject();
...
Z. 382: $page = self::getPageObject(rex::getProperty('start_page'));

Aufgerufen aus redaxo/src/core/backend.php 
Zeile 216: rex_be_controller::checkPagePermissions($user)